src/threadNames.cpp

```diff
@@ -0,0 +1,52 @@
+#include "threadNames.h"
```

Comment:
New source files should include standard license header.

Reply:
OK. Who should be the copyright owner for new files? 
While it is told 
> You _must_ cause any modified files to carry prominent notices stating that You changed the files; and ...

> You _may_ add Your own copyright statement to Your modifications and ...

It seems strange for me to be an owner for two files in you project, though.

Reply:
The above statements are about redistribution.
When contributing changes, you typically grant non-exclusive copyright to the project owner. I'll formulate contribution instructions in a document a bit later.

So, please leave ` * Copyright 2018 Andrei Pangin` line in the header. If you wish, you *may* add an extra copyright line with your name.